.github/workflows/e2e.yml

```diff
@@ -0,0 +1,80 @@
+name: Test e2e
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - develop
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
+    env:
+      FIFTYONE_DO_NOT_TRACK: true
+      ELECTRON_EXTRA_LAUNCH_ARGS: "--disable-gpu"
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip setuptools wheel
+
+      - name: Cache Node Modules
+        id: node-cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            app/node_modules
+            app/.yarn/cache
+          key: node-modules-${{ hashFiles('app/yarn.lock') }}
+
+      - name: Install app
+        if: steps.node-cache.outputs.cache-hit != 'true'
+        run: cd app && yarn install
+
+      - name: Build app
+        run: make app
+
+      - name: Install fiftyone
+        run: pip install .
+
+      - name: Configure
+        id: test_config
+        run: |
+          python tests/utils/setup_config.py
+          python tests/utils/github_actions_flags.py
+
+      - name: FFmpeg
+        uses: FedericoCarboni/setup-ffmpeg@v2
+
+      - name: Run e2e tests
```

Comment:
The test step is currently passing when tests fail. Ideally this will fail when tests fail so the workflow calls can then fail

Reply:
I saw you made the changes, were you able to test it?